docs/index.rst

```diff
@@ -0,0 +1,48 @@
+casa-formats-io documentation
+=============================
+
+Scope
+-----
+
+The **casa-formats-io** package is a small package which implements
+functionality to read data stored in CASA formats (such as .image datasets).
+This implementation is independent of and does not use `casacore
+<https://casacore.github.io/casacore/>`_. The motivation for this package is
+to provide:
+
+* Efficient data access via `dask <https://dask.org/>`_ arrays
+* Cross-platform data access, supporting Linux, MacOS X and Windows
+* Data access with all modern Python versions, from 3.6 to the latest Python version
+
+At this time, only reading .image datasets is supported. Reading measurement sets
+(.ms) or writing data of any kind are not yet supported.
+
+Using casa-formats-io
+---------------------
+
+To construct a dask array backed by a .image dataset, use the
+:func:`~casa_io_formats.image_to_dask` function::
+
+    >>> from casa_formats_io.casa_dask import image_to_dask
+    >>> image_to_dask('my_dataset.image/')
+    dask.array<CASA Data 6bd6f684-0d21-4614-b953, shape=(2114, 1, 2450, 2450), dtype=float32, chunksize=(14, 1, 350, 2450), chunktype=numpy.ndarray>
+
+Note that rather than use the native CASA chunk size as the size of dask chunks,
+which is extremely inefficient for large datasets (for which there may be a
+million CASA chunks or more), the `casa_io_formats.image_to_dask` function will
+automatically join neighbouring chunks together on-the-fly which then provides
+significantly better performance.
+
+In addition to :func:`~casa_io_formats.image_to_dask`, this package
+implements :func:`~casa_formats_io.getdesc` and :func:`~casa_formats_io.getdmino`
```

Comment:
```suggestion
implements :func:`~casa_formats_io.getdesc` and :func:`~casa_formats_io.getdminfo`
```
Link to https://casadocs.readthedocs.io/ ?

Reply:
https://casadocs.readthedocs.io/en/stable/api/tt/stubs.tools.table.html?highlight=getdminfo#stubs.tools.table.getdminfo
https://casadocs.readthedocs.io/en/stable/api/tt/stubs.tools.table.html?highlight=getdminfo#stubs.tools.table.getdesc

Reply:
Done, thanks for the links!